src/Migration/ConnectionProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections.Generic;
+
+namespace RapidCore.Mongo.Migration
+{
+    public class ConnectionProvider : IConnectionProvider
+    {
+        protected readonly IDictionary<string, MongoDbConnection> connections = new Dictionary<string, MongoDbConnection>();
+        
+        public MongoDbConnection Default()
+        {
+            return Named("UseAsDefault");
+        }
+
+        public MongoDbConnection Named(string name)
+        {
+            return connections[name.ToLower()];
```

Comment:
We should use `ToLowerInvariant()` to avoid culture settings mucking it up.
More detains here:
https://stackoverflow.com/a/19778131/1662254